lib/bolt/cli.rb

```diff
@@ -460,6 +484,8 @@ def execute(options)
             end
         end
 
+        failures = count_failures(results)
+        @logger.notice { log_summary(mode, object, results.size, failures) }
```

Comment:
The human outputter prints this by default so it should be at info level to avoid duplication

Reply:
They're slightly different outputs, "Ran task exec on 2 nodes with 0 failures" and "Ran on 2 nodes in 0.72 seconds" -- agreed we don't need both, but not sure if one is preferred or if they should be combined or whatnot.